docs/implementation-references/getting-started-guide.md

```diff
@@ -190,15 +257,17 @@ TO0 Response Wait for <guid> : 3600
 TO0 Client finished for GUID <guid>
 ```
 
-***NOTE:***
+**After the completion of TO0, RV Server having an entry in a table that associates the Device GUID with the Owner Address. During T01, the device collects this OwnerAddress from RV and uses this OwnerAddress to contact Owner for TO2 and completes onboarding.**
+
```

Comment:
here it doesnt complete onboarding right ?

Reply:
Nope.

Reply:
then should we call it as completes onboarding ?